README.rst

```diff
@@ -211,75 +211,92 @@ To make ``--follow-tags`` permanent run this command: ::
 
 
 
-Building RPMs on travis-ci.com
+Building RPMs on github.com
 ------------------------------
 
-`travis-ci.com <https://travis-ci.com>`_ automatically builds RPMs and uploads
+`github.com <https://github.com/features/actions>`_ automatically builds RPMs and uploads
 them to ``packages.nethserver.org``, if configured with enough environment variables
 and upload secrets.
 
 Configuration
 ^^^^^^^^^^^^^
 
-To automate the RPM build process using Travis CI
+To automate the RPM build process using GitHub
 
-* create a ``.travis.yml`` file inside the source code repository hosted on
+* create a ``.github/workflows/make-rpms.yml`` file inside the source code repository hosted on
   GitHub.
 
-* the repository must have Travis CI builds enabled and upload secrets properly set up.
+* the repository must have builds enabled and upload secrets properly set up, if the repo is inside Nethserver org, everything is already set up (as long as you have write access to the repo).
```

Comment:
we can also add that the organization admin must allow access to relative the organization's secrets from the package repository.